tests/bug-2917/test.yaml

```diff
@@ -0,0 +1,215 @@
+requires:
+  min-version: 7
+
+pcap: false
+
+args:
+  - --engine-analysis
+
+checks:
+  - filter:
+      filename: rules.json
+      count: 1
+      match:
+        app_proto: http
```

Comment:
I find this sorting very hard to read. The default output order should ideally be preserved here

Reply:
Sorry. Didn't realize I did that.